Snippets/ASQ/ASQ_9/Usage.cs

```diff
@@ -0,0 +1,161 @@
+﻿// ReSharper disable SuggestVarOrType_Elsewhere
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Queues;
+using Microsoft.Azure.Cosmos.Table;
+using NServiceBus;
+
+[SuppressMessage("ReSharper", "UnusedMember.Local")]
+class Usage
+{
+    void UseTransport(EndpointConfiguration endpointConfiguration)
+    {
+        #region AzureStorageQueueTransportWithAzure
+
+        var transport = endpointConfiguration.UseTransport<AzureStorageQueueTransport>();
+        transport.ConnectionString("DefaultEndpointsProtocol=https;AccountName=[ACCOUNT];AccountKey=[KEY];");
+
+        #endregion
+    }
+
+    void CodeOnly(EndpointConfiguration endpointConfiguration)
+    {
+        var queueServiceClient = new QueueServiceClient("connectionString", new QueueClientOptions());
+        var blobServiceClient = new BlobServiceClient("connectionString", new BlobClientOptions());
+        var cloudStorageAccount = CloudStorageAccount.Parse("connectionString");
+        var cloudTableClient = cloudStorageAccount.CreateCloudTableClient();
+
+        #region AzureStorageQueueConfigCodeOnly
+
+        var transport = endpointConfiguration.UseTransport<AzureStorageQueueTransport>();
+        transport.ConnectionString("DefaultEndpointsProtocol=https;AccountName=[ACCOUNT];AccountKey=[KEY];");
+        transport.BatchSize(20);
+        transport.MaximumWaitTimeWhenIdle(TimeSpan.FromSeconds(1));
+        transport.DegreeOfReceiveParallelism(16);
+        transport.PeekInterval(TimeSpan.FromMilliseconds(100));
+        transport.MessageInvisibleTime(TimeSpan.FromSeconds(30));
+        transport.UseQueueServiceClient(queueServiceClient);
+        transport.UseBlobServiceClient(blobServiceClient);
+        transport.UseCloudTableClient(cloudTableClient);
+
+        #endregion
+    }
+
+    async Task SendToMulitpleAccountUsingConnectionSTring(IEndpointInstance endpointInstance)
+    {
+        #region storage_account_routing_send_options_full_connectionstring
+
+        await endpointInstance.Send(
```

Comment:
I thought we no longer support this and enforce alias always except we allow receive from previous versions

Reply:
The page that uses this snippet is the only one that has not been completed. @lailabougria is going to work on it. See TF channel.